packages/utilities/psammead-storybook-helpers/src/withServicesKnob.jsx

```diff
@@ -2,20 +2,41 @@ import React from 'react';
 import { Helmet } from 'react-helmet';
 import { select } from '@storybook/addon-knobs';
 import * as scripts from '@bbc/gel-foundations/scripts';
-import LANGUAGE_VARIANTS from './text-variants';
+import EXAMPLES from './text-variants';
+
+const DEFAULT_SERVICE = 'news';
+const SERVICES_LIST = Object.keys(EXAMPLES);
 
 export default ({
-  defaultService = 'news',
-  services = Object.keys(LANGUAGE_VARIANTS),
+  defaultService = DEFAULT_SERVICE,
+  services = SERVICES_LIST,
 } = {}) => storyFn => {
-  const includesService = service => services.includes(service);
-  const service = select(
+  const getServiceAndVariant = selectedItem =>
```

Comment:
Slightly misleading function name as it may not return a variant. The name suggests this function should be split up anyway. Maybe a function called `getService` and then `getVariant`.